### PR TITLE
Triggers: Adjust SetEventUi & SetMesh delay, implement SetState/UseState

### PR DESCRIPTION
**Pull request by Zintixx**
- Also some misc renaming on some trigger functions.

### CODE REVIEW

File: Maple2.Model/Enum/BannerType.cs
Comment (AngeloTadeucci) [inline Maple2.Model/Enum/BannerType.cs:1]: delete file right?

**Review by AngeloTadeucci — APPROVED** [review]

File: Maple2.File.Ingest/Utils/PyParameter.cs [hunk 1]
```diff
@@ -134,6 +137,18 @@ internal record PyParameter(ScriptType Type, string Name) {
                 "None" => "Weather.Clear",
                 _ => $"Weather.{TriggerTranslate.ToPascalCase(value)}",
             },
+            ScriptType.EnumBannerType => string.IsNullOrWhiteSpace(value) ? null : value switch {
+                // The type values intentionally do not match the BannerType enum values.
+                //"0" => "",
+                "1" => "BannerType.Text",
+                //"2" => "",
+                "3" => "BannerType.Winner",
```
Comment (kOchirasu) [inline Maple2.File.Ingest/Utils/PyParameter.cs:145]: Winner/Loser or Win/Lose feels more consistent

Comment (kOchirasu) [reply Maple2.File.Ingest/Utils/PyParameter.cs:145]: if it's to match text that's fine too

Comment (Zintixx) [reply Maple2.File.Ingest/Utils/PyParameter.cs:145]: yeah the banners actually say "Winner" and "Lose"

**Pull request merged**